src/views/credits/credits.jsx

```diff
@@ -185,125 +187,29 @@ var Credits = React.createClass({
-                    other universities, including Yasmin Kafai (who collaborated on the
-                    <a href="http://www.nsf.gov/awardsearch/showAward?AWD_ID=0325828"> initial NSF Scratch grant</a>
-                    ) at the University of Pennsylvania Graduate School of Education, Karen Brennan (who leads the
-                    <a href="http://scratched.gse.harvard.edu/"> ScratchEd project</a>) at the Harvard Graduate School
-                    of Education, Benjamin Mako Hill at the University of Washington, Andrés Monroy Hernández at
-                    Microsoft Research, Mimi Ito and Crystle Martin at the University of California, Irvine, Quinn
-                    Burke at College of Charleston, Deborah Fields at Utah State University, and Kylie Peppler at
-                    Indiana University.
-                </p>
-
-                <h2>Acknowledgements</h2>
-                <p>
-                    The following people have also contributed to the development and support of Scratch over the
-                     years: Susan Abend, Robbie Berg, Lauren Bessen, Keith Braadfladt, Susan Carillo, Will Denton,
-                     Nathan Dinsmore, Catherine Feldman, Jodi Finch, Ioana Fineberg, Rachel Garber, Chris Garrity,
-                     Cassy Gibbs, Brian Harvey, Roland Hebert, Tracy Ho, Benjamin Howe, Kapaya Katongo, Evan Karatzas,
-                     Christine Kim, Joren Lauwers, Mike Lee, Jeff Lieberman, Mark Loughridge, Kelly Liu, Anthony Lu,
-                     Danny Lutz, David Malan, Wayne Marshall, John McIntosh, Paul Medlock-Walton, Dongfang (Tian) Mi,
-                     Ximena Miranda, Jens Moenig, Evan Moore, Geetha Narayanan, Kate Nazemi, Liddy Nevile, Wing Ngan,
-                     Derek O&#39;Connell, Tim Radvan, Karen Randall, Ian Reynolds, Miriam Ruiz, Chinua Shaw, Ed Shems,
-                     Cynthia Solomon, Daniel Strimpel, Kilmer Sweazy, John Henry Thompson, Ubong Ukoh, Vladimir Vuksan,
-                     Han Xu, and the many volunteer <a href="http://wiki.scratch.mit.edu/wiki/Translators"> Scratch
-                     Translators</a> from around the world.
-                </p>
-
-                <p>
-                    We greatly appreciate all of the contributions by members of the worldwide Scratch community,
-                     who have shaped the direction of Scratch by sharing their projects, comments, and ideas.
-                </p>
-                <p>
-                    The ideas of Seymour Papert and Alan Kay have deeply inspired and influenced our work on Scratch.
-                </p>
-
-                <h2>Supporting Organizations</h2>
-                <p>
-                    The following organizations have provided major financial support for Scratch:
-                </p>
-                <p>
-                    <a href="http://www.nsf.gov/">National Science Foundation</a>,
```

Comment:
For all of these links with organizations, we don't want to add them to the `intl.json` file – they're proper nouns, so I think we want to keep them untranslated